VMEncryption/main/common_parameters.json

```diff
@@ -1,4 +1,4 @@
 {
-    "extension_version": "1.1.0.48",
+    "extension_version": "1.1.0.49",
```

Comment:
Note:  I believe this is correct but just to make sure we are clear - this is the same version that is currently in deployment and staged in the release branch (since deployment has not been completed, master may still be one behind)